texar/torch/modules/tokenizers/pretrained_bert_tokenizer_utils.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils of pre-trained BERT tokenizer.
+"""
+
+from typing import Dict, List, Optional
+
+import collections
+import unicodedata
+
+
+__all__ = [
+    "load_vocab",
+    "BasicTokenizer",
+    "WordpieceTokenizer",
+]
+
+
+def load_vocab(vocab_file: str) -> Dict[str, int]:
+    r"""Loads a vocabulary file into a dictionary."""
+    vocab: Dict[str, int] = collections.OrderedDict()
+    with open(vocab_file, "r", encoding="utf-8") as reader:
+        tokens = reader.readlines()
+    for index, token in enumerate(tokens):
+        token = token.rstrip('\n')
+        vocab[token] = index
+    return vocab
+
+
+class BasicTokenizer:
```

Comment:
Are these tokenizers exposed to users? If so, they should be modules (inheriting `ModuleBase`), and expose `forward` (instead of `tokenize(..)`) with the same interfaces as other pretrained tokenizers